content/en/docs/concepts/workloads/pods/_index.md

```diff
@@ -14,12 +14,11 @@ card:
 
 _Pods_ are the smallest deployable units of computing that you can create and manage in Kubernetes.
 
-A _Pod_ (as in a pod of whales or pea pod) is a group of one or more
-{{< glossary_tooltip text="containers" term_id="container" >}}, with shared storage and network resources, and a specification for how to run the containers. A Pod's contents are always co-located and
+A _Pod_ (as in a pod of whales or pea pod) is a cluster of
```

Comment:
Why replace "group" with "cluster" here?

Reply:
Because I think "cluster" is more a technical term than "group", and they both capture the essence of multiple containers being brought together. Also, the term "cluster" aligns well with "pod" as it is associated with the idea of grouping.

Reply:
Hi @RakshitKumar04 , Group would be a better option here because a pod might contain 2 or more containers that may or may not be of same type ( in terms of image they are using Ex. redis, ngnix etc.), we say kubernetes cluster because Nodes have similiar properties like their components - kubelet , kube-poxy (in case of multiple worker nodes).

Reply:
ok

Reply:
Hey, @utkarsh-singh1 can you please suggest any other improvement in the Pod description, to resolve the issue.